### PR TITLE
[5.3] Create option for disable logging

### PR DESCRIPTION
**Pull request by driade**
Add an option to completely disable logging, setting the option "log" to "null" at config.

If accepted, docs and stub should be updated.

Thanks.

### CODE REVIEW

File: src/Illuminate/Contracts/Logging/Log.php [hunk 1]
```diff
@@ -95,4 +95,11 @@ public function useFiles($path, $level = 'debug');
      * @return void
      */
     public function useDailyFiles($path, $days = 0, $level = 'debug');
+
+    /**
+     * Register a null log handler (disable logging)
+     *
+     * @return void
+     */
+    public function useNull();
```
Comment (GrahamCampbell) [inline src/Illuminate/Contracts/Logging/Log.php]: You cannot add this method to the contract. It'll break existing code such as Bugsnag.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by driade** [comment]
Understood, removed from contract, though I'm not certain whether it should be this way or added to the contract and appointed to 5.4. The way it's at this moment would let the option only available to Monolog, which should be fine anyway.

Would you suggest a "global" null handler for all the loggers?

Thanks.

**Comment by GrahamCampbell** [comment]
There are other ways to disable logging that don't require modifications to the code. I don't think this needs merging tbh.

**Comment by driade** [comment]
Closed then.

**Pull request closed without merging**